FILES/N_0x10144267dd22866c.md

```diff
@@ -1,29 +1,22 @@
 ---
 ns: FILES
+aliases: ["_0x10144267DD22866C"]
```

Comment:
The alias shouldn't have the underscore.

Ideally, it should look like this:
`aliases: ["0x10144267DD22866C"]`

Reply:
Done, ty!